Bigtable/src/Instance.php

```diff
@@ -0,0 +1,313 @@
+     * @codingStandardsIgnoreEnd
+     *
+     * @param array $clusterMetadataList [
+     *        $bigtable->clusterMetadata('clusterId1', 'locationId1', 'SSD', 3),
+     *        $bigtable->clusterMetadata('clusterId2', 'locationId2', 'SSD', 3)
+     * ]
+     * @param string $displayName **Defaults to** the value of $instanceId.
+     * @param array $labels as key/value pair ['foo' => 'bar']. For more information, see
+     *        [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
+     * @param int $type Possible values are represented by the following constants:
+     *        `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_PRODUCTION`,
+     *        `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT` and
+     *        `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     *        **Defaults to** using `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     *
+     * @return LongRunningOperation<Instance>
+     */
+    public function create(
+        array $clusterMetadataList,
+        $displayName = null,
+        array $labels = [],
+        $type = self::INSTANCE_TYPE_UNSPECIFIED
+    ) {
+        if (empty($clusterMetadataList)) {
+            throw new \InvalidArgumentException('At least one clusterMetadata must be passed');
+        }
+        $projectName = InstanceAdminClient::projectName($this->projectId);
+        $displayName = ($displayName) ? $displayName : $this->id;
+
+        $clustersArray = [];
+        foreach ($clusterMetadataList as $value) {
+            if (!isset($value['clusterId'])) {
+                throw new \InvalidArgumentException("Cluster id must be set");
+            }
+            $this->validate($value['clusterId'], 'cluster');
+            $clusterId = $value['clusterId'];
+
+            if (!isset($value['locationId'])) {
+                throw new \InvalidArgumentException("Location id must be set");
+            }
+            $this->validate($value['locationId'], 'location');
+            $locationId = $value['locationId'];
+
+            $value['location'] = InstanceAdminClient::locationName($this->projectId, $locationId);
+
+            $value['defaultStorageType'] = isset($value['storageType'])
+                ? $value['storageType']
+                : self::STORAGE_TYPE_UNSPECIFIED;
+
+            $clustersArray[$clusterId] = $value;
+        }
+
+        $operation = $this->connection->createInstance([
+            'parent' => $projectName,
+            'instanceId' => $this->id,
+            'instance' => [
+                'displayName' => $displayName,
+                'type' => $type,
+                'labels' => $labels
+            ],
+            'clusters' => $clustersArray
+        ]);
+
+        return $this->resumeOperation($operation['name'], $operation);
```

Comment:
Now that we have undocumented the `resumeOperation` method, how do we expect users to know they can resume an active operation? What if they need to finish polling the operation in another process?